init-buildx.sh

```diff
@@ -14,10 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -o errexit -o nounset -o pipefail
+set -o errexit
+set -o nounset
+set -o pipefail
 
 export DOCKER_CLI_EXPERIMENTAL=enabled
-
+echo "set docker cli experimental"
 # Ensure we use a builder that can leverage it (the default on linux will not)
 docker buildx rm windows-builder || true
+
+echo "buildx"
```

Comment:
is this just for debugging or do you want to merge this? If you want to merge this, can you make the statements more specific?

Reply:
this just for debugging. Some issue with set pipefail

Reply:
I have to remove pipefail, it always return error: invalid option.